### DISCUSSION TITLE
Scatter Chart with dynamic series data

### DISCUSSION THREAD

**Discussion by thiagoolsilva** [jxlsteam/jxls#277] [Q&A]
I'd love to get more information on creating a scatter chart with dynamic data using jxls 2.X version. I did some research, but I've found the best solution to solve my problem.

I've used the code below to create the following template containing dynamic series.

         { 
            Context context = new Context();
            context.putVar("employee", contents);

            JxlsHelper.getInstance().processTemplate(is, os, context);

        }

With the POJO

```
public class Employee {
    private String name;
    private List<Integer> x;
    private List<Integer> y;
}

```
Template
[chart_template.xlsx](https://github.com/jxlsteam/jxls/files/13572982/chart_template.xlsx)
Excel output
[chart_template_output.xlsx](https://github.com/jxlsteam/jxls/files/13573019/chart_template_output.xlsx)

Thanks for your support

**Comment by SoltauFintel** [comment]
What's the question? Please add an Excel file with the expectation.

**Comment by thiagoolsilva** [comment]
Thanks for your feedback. I'd love to create a chart Scatter type using the library JXLS 2.X using dynamic time series inserted in xlsx file from POJO java class. Please check the above files attached to this task to get more information about how I'm doing the use of the library to achieve this goal. 

Thanks!

**Reply by SoltauFintel** [reply]
I have studied your request carefully.

Have you ever tried to create the result manually (without JXLS)? Is that possible?

Without seeing the expected file, I can't help. I have no knowledge for this chart type.
